### ISSUE TITLE
Pagesize select control in Datagrid footer do not have label announcing by screen reader in accessibility test

### ISSUE THREAD

**Issue by xuelichao** [vmware/clarity#6437]
## Describe the bug
1. When enable [clrDgPageSize]="20" and [clrPageSizeOptions]="[10,20,50,100]", it will generate a select control for clrPageSizeOptions.
2. The screen reader can not get any label content for helping user Due to there is no label for clrPageSizeOptions select, 

## How to reproduce
1. Any datagrid control with pagesize enabled 
2.     <clr-dg-footer>
        <clr-dg-pagination #pagination [clrDgPageSize]="20" [clrDgTotalItems]="items.totalNum"
          [clrPageSizeOptions]="[10,20,50,100]">

_Please provide a link to a reproduction scenario using one of the [Clarity Stackblitz templates](https://stackblitz.com/@clr-team). Reports without a clear reproduction may not be prioritized until one is provided._

1.  Go to '...'
2.  Click on '....'
3.  Scroll down to '....'
4.  See error

## Expected behavior

Gatagrid should support to configure a label for clrPageSizeOptions select or a way to configure aria-label attribute.

_A clear and concise description of what you expected to happen._

## Versions
4.0.8

**Clarity project:**

- [* ] Clarity Core
- [ *] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [*] v4.x
- [ ] v5.x

**Framework:**

- [*] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
_ie: Angular 11_

**Device:**

- Type: [e.g. MacBook] MacBook
- OS: [e.g. iOS] IOS
- Browser [e.g. chrome, safari] chrome

## Additional notes

_Add any other notes about the problem here._

**Comment by Jinnie** [comment]
It's a known accessibility issue, which we track internally, as we do with all a11y bugs for better collaboration with our a11y team.

In this case we suggest the following workaround.
A directive may be used to add the ID to the select:
https://stackblitz.com/edit/datagrid-v4-template-ees6hm?file=src%2Fapp%2Fa11y-page-options.directive.ts

Generally this is not a good pattern, as it relies on internally generated markup structure, but with Clarity Angular frozen for breaking changes, it is a viable workaround.

**Comment by xuelichao** [comment]
https://stackblitz.com/edit/datagrid-v4-template-ees6hm?file=src%2Fapp%2Fa11y-page-options.directive.ts
The above directive is workable for me.

Thanks
